usr/include/stdll.h

```diff
@@ -424,6 +429,8 @@ struct ST_FCN_LIST {
     ST_C_CancelFunction ST_CancelFunction;
 
     ST_C_IBM_ReencryptSingle ST_IBM_ReencryptSingle;
+
+    ST_C_HandleEvent ST_HandleEvent;
```

Comment:
C_IBM_HandleEvent ? Its a vendor defined extension.

Reply:
Well, no, it is not an external API function. There is no C_HandleEvent function that an external application could call. However, its a function that a token must provide in order to implement event handling. It is called from the API layer (where the event thread runs).

To be consequent I could actually remove the '_C_' then, since it does not correspond to an external API function.... However, all the functions in struct ST_FCN_LIST have a '_C_' in its names, even though they have different signatures than the corresponding external C_xx functions. 

What do you think ?

Reply:
> Well, no

Ah, yeah, i see. Right, the C_ is quite confusing. Is the order of the members in ST_FCN_LIST important ? If not, id just add a comment above saying that below functions are not part of the API.

Reply:
No the order is not important. So I'll add a comment as you suggested.